Makefile

```diff
@@ -14,16 +14,16 @@ build:
 .PHONY: build-full
 build-full:
 	docker build -t $(DOCKER_IMAGE) --target production -f ./server/conf/docker/Dockerfile .
-	docker tag $(DOCKER_IMAGE) $(DOCKER_IMAGE_LATEST)
-	docker tag $(DOCKER_IMAGE) $(DOCKER_IMAGE_BRANCH)
+	# docker tag $(DOCKER_IMAGE) $(DOCKER_IMAGE_BRANCH) # Note: Currently broken for branches with slashes
```

Comment:
**Question.** Is commenting this out an innocuous quick fix?

Reply:
Somewhat. I could remove it outright. I'd like to get branch deploys working, but we need a working docker tag scheme for github branches